test/micro/org/openjdk/bench/jdk/incubator/vector/Float16OperationsBenchmark.java

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights vectorReserved.
```

Comment:
```suggestion
 * Copyright (c) 2025, Oracle and/or its affiliates. All rights Reserved.
```
Looks like a "vector" snuck in there :wink:

Reply:
DONE